dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/GetOrderStatusInfoByIdWithApiKeySpec.scala

```diff
@@ -0,0 +1,166 @@
+package com.wavesplatform.it.matcher.api.http
+
+import com.softwaremill.sttp.StatusCodes
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.api.http.entities.HttpOrderBookHistoryItem
+import com.wavesplatform.dex.domain.account.KeyPair.toAddress
+import com.wavesplatform.dex.domain.bytes.codec.Base58
+import com.wavesplatform.dex.domain.order.Order
+import com.wavesplatform.dex.domain.order.OrderType.{BUY, SELL}
+import com.wavesplatform.dex.it.api.RawHttpChecks
+import com.wavesplatform.dex.it.docker.apiKey
+import com.wavesplatform.dex.model.{AcceptedOrderType, OrderStatus}
+import com.wavesplatform.it.MatcherSuiteBase
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class GetOrderStatusInfoByIdWithApiKeySpec extends MatcherSuiteBase with TableDrivenPropertyChecks with RawHttpChecks {
+
+  override protected def dexInitialSuiteConfig: Config = ConfigFactory.parseString(
+    s"""waves.dex {
+       |  price-assets = [ "$BtcId", "$UsdId", "WAVES" ]
+       |}""".stripMargin
+  )
+
+  val order = mkOrder(alice, wavesUsdPair, BUY, 10.waves, 2.usd)
+
+  override protected def beforeAll(): Unit = {
+    wavesNode1.start()
+    broadcastAndAwait(IssueBtcTx, IssueUsdTx)
+    dex1.start()
+  }
+
+  "GET /matcher/orders/{address}/{orderId} " - {
+
+    def httpOrderInfo(o: Order, s: OrderStatus, f: Long = 0.003.waves, awp: Long = 0, tepa: Long = 0): HttpOrderBookHistoryItem =
+      HttpOrderBookHistoryItem(
+        o.id(),
+        o.orderType,
+        AcceptedOrderType.Limit,
+        o.amount,
+        s.filledAmount,
+        o.price,
+        f,
+        s.filledFee,
+        o.feeAsset,
+        o.timestamp,
+        s.name,
+        o.assetPair,
+        awp,
+        o.version,
+        tepa
+      )
+
+    "should return correct status of the order" in {
+      val o = mkOrder(alice, wavesUsdPair, BUY, 10.waves, 2.usd)
+      placeAndAwaitAtDex(o)
+
+      withClue(" - accepted") {
+        validate200Json(dex1.rawApi.getOrderStatusInfoByIdWithApiKey(alice, o.id(), Some(alice.publicKey))) should be(httpOrderInfo(
```

Comment:
Let's replace `should be` with `should matchTo` in such places, because `matchTo` shows only the difference

Reply:
ok